handbook/engineering/code-intelligence/goals.md

```diff
@@ -14,27 +14,28 @@ Progress on adoption and usage is tracked in our [Looker dashboard](https://sour
 
 **Outcome:**
 
-- 25 popular open source C++ repositories are indexed on Sourcegraph Cloud.
-- 25 popular open source Java repositories are indexed on Sourcegraph Cloud.
+- 30 popular open source C++ repositories are indexed on Sourcegraph Cloud.
+- 30 popular open source Java repositories are indexed on Sourcegraph Cloud.
 - C++ and Java indexers are adopted by at least 3 customers each.
 - Precise code intel reaches [*N<sub>0</sub>*][N0] precise code intel operations a month.
 - CE and Sales have a significant amount of open source repos to showcase and can easily demonstrate the value of setting up precise code intelligence.
 - For languages that are particularly subject to unusual setups and tooling, our indexed open source showcase helps us determine if setup issues are caused by an unusual customer dev environment.
 
 **Milestones:**
 
-1. 🔄 Sourcegraph Cloud has up-to-date precise code intelligence for C++ repositories.
-    - 🔄 Increase observability in the code intel executor to make troubleshooting failed auto-index records possible
-    - 🔄 Auto-index 5 repositories on Sourcegraph Cloud
+1. ✅ Sourcegraph Cloud has up-to-date precise code intelligence for C++ repositories.
+    - ✅ Increase observability in the code intel executor to make troubleshooting failed auto-index records possible
+    - ✅ Auto-index 5 repositories on Sourcegraph Cloud
 
-1. Sourcegraph Cloud has up-to-date precise code intelligence for C++ repositories that use cmake as a build system. 
-    - Auto-detect common cmake project configurations in the auto-indexer
-    - Auto-index 5 repositories on Sourcegraph Cloud that use cmake as a build system
+1. 🔄 Sourcegraph Cloud can auto-configure precise code intelligence for C++ repositories that use cmake as a build system. 
+    - 🔄 Auto-index 5 repositories on Sourcegraph Cloud that use cmake as a build system
+    - Add inference rules for cmake projects and have at least 5 repositories fully auto-indexed on Cloud
 
-1. Sourcegraph Cloud has up-to-date precise code intelligence for C++ repositories that use Bazel as a build system.
+1. Sourcegraph Cloud can auto-configure precise code intelligence for C++ repositories that use Bazel as a build system.
     - Support Bazel projects in lsif-clang
-    - Auto-detect common Bazel project configurations in the auto-indexer
     - Auto-index 5 repositories on Sourcegraph Cloud that use Bazel as a build system
+    - Add inference rules for Bazel projects and have at least 5 repositories fully auto-indexed on Cloud
```

Comment:
I'm not sure if we've had a different experience while I've been gone, but the vast majority of projects I've encountered (Bazel or CMake) has required some bespoke step for that project which could only be derived from a README or experimentation. So I guess I never saw auto-configured indexing as a milestone for C++ since it's unclear that it'll really be possible for more than a small slice of projects. Happy to be wrong about this though!

Reply:
It's fine for it to be unclear at the moment, our milestones have a big piece of experimental work and validating these hypotheses will be part of it. We can always modify our roadmap and milestones once we have a clearer view of what will or won't be possible or if we prove that the amount of added value isn't worth the effort.